README.md

```diff
@@ -35,4 +35,6 @@ pvalue(SignedRankTest(x))
 
 ## Documentation
 
-Full documentation available at [Read the Docs](http://hypothesistestsjl.readthedocs.org/en/latest/index.html).
+Full documentation available at
```

Comment:
While you're at it, better adopt the standard layout used by other packages. See for example the one for StatsBase: https://raw.githubusercontent.com/JuliaStats/StatsBase.jl/master/README.md

Reply:
Done.